source/documentation/publish_and_manage_data/get_and_manage_accounts.md

```diff
@@ -1,34 +1,32 @@
 # Get and manage accounts
 
-To publish and manage datasets on data.gov.uk you must have an administrator (admin) or editor account.
+There are two levels of access on data.gov.uk: editor and administrator (admin).
 
-Each account is associated with an organisation responsible for the dataset. For example, a local authority or central government department.
+Editors are able to publish and manage datasets associated with their organisation (e.g. a local authority or central government department). Editors and admins can be in multiple organisations.
 
```

Comment:
Style point: We use 'for example' in full rather than 'eg'. So we can change the text in brackets to '(for example, a local authority or central government department)'.